sechub-doc/src/docs/asciidoc/documents/operations/server/server_deployment.adoc

```diff
@@ -151,6 +151,14 @@ accidently add the credentials to your repository.
 ====
 
 ====== Logging
+*Default output*
 
 {sechub} uses per default logging to stdout, so you just need to use something like
-`fluentd` PODs to gather log output.
\ No newline at end of file
+`fluentd` PODs to gather log output.
+
+*Logstash support*
+
+When you want to use `logstash` (e.g. when using `Kibana`) you just have to set the environment variable 
```

Comment:
Kibana is just a web front-end for ElasticSearch. [OpenSearch](https://www.opensearch.org/) is a fork of ElasticSearch.

```suggestion
When you want to use `logstash` (e.g. when using `ElasticSearch` or `OpenSearch`) you just have to set the environment variable 
```